plugins/VstEffect/VstEffect.cpp

```diff
@@ -86,15 +86,11 @@ bool VstEffect::processAudioBuffer( sampleFrame * _buf, const fpp_t _frames )
 	if( m_plugin )
 	{
 		const float d = dryLevel();
-#ifdef __GNUC__
-		sampleFrame buf[_frames];
-#else
-		sampleFrame * buf = new sampleFrame[_frames];
-#endif
-		memcpy( buf, _buf, sizeof( sampleFrame ) * _frames );
+		auto buf = std::vector<sampleFrame>{static_cast<size_t>(_frames)};
+		memcpy(buf.data(), _buf, sizeof(sampleFrame) * _frames);
```

Comment:
Realtime safety again. Original code is also not good when `__GNUC__` is not defined, and should be changed to use `_alloca` instead, but is probably out of scope.